flight/Modules/Attitude/attitude.c

```diff
@@ -1489,6 +1483,9 @@ static void updateNedAccel()
 	accel_ned[2] += GRAVITY;
 	
 	NedAccelData accelData;
+	
+	/* XXX this is redundant with another set when in INS modes... */
+	NedAccelGet(&accelData);
```

Comment:
This is still wrong, and there's a bug open for it.  But at least it's not tainted by uninit'd stuff in this path.